training/v5p/DeepSeek3-671B-MaxText/README.md

```diff
@@ -0,0 +1,61 @@
+# Instructions for training DeepSeek-671B-MaxText on TPU v5p-1024
+
+## XPK setup
+Please follow this [link](https://github.com/gclouduniverse/reproducibility/tree/main/Training/TPU-v5p/XPK_README.md) to create your GKE cluster with XPK
```

Comment:
These XPK instructions are out of date. Can you please replace them here and in the other two files with [the ones for Trillium](https://github.com/gclouduniverse/reproducibility/tree/main/Training/TPU-v5p/XPK_README.md)?

Maybe just updating that file you already have linked would be the best way for now